app/models/sales_log.rb

```diff
@@ -21,13 +21,13 @@ def self.editable_fields
   end
 
   def form_name
-    return unless saledate
+    return unless startdate
```

Comment:
Isn't the field called `saledate` on sales logs still?

Reply:
Tyvm, it didn't fail cause there was a function that returns saledate for startdate! Will change it here tho